build.gradle

```diff
@@ -57,6 +57,13 @@ allprojects {
 
 	group = "net.fabricmc.fabric-api"
 
+	sourceSets {
+		testmod {
+			compileClasspath += main.compileClasspath
```

Comment:
Hmm you declared another compile configuration below
```suggestion
			compileClasspath += main.compileClasspath + main.output
```
~~Should compile against the rest of the main code as well~~

Reply:
Whats the use of this?

Reply:
not really useful as you already have implementation configuratiom